Ianis/task2/ok.rb

```diff
@@ -0,0 +1,47 @@
+require "watir"
+
+class Odnoklassniki
+
+  attr_reader :browser
+  def initialize
+    @browser = Watir::Browser.new :chrome
+    browser.goto("google.com")
+    sleep(1)
+  end
+
+  def collect_links
+    search_person
+    return unless present_photo?
+    take_photo
+    { links: take_links }
+  end
+
+  def search_person
+    browser.text_field(id: "lst-ib").set("Иван Петров" + " одноклассники")
+    browser.input(name: "btnK").click
+    sleep(1)
+    browser.div(class: "srg").links[0].click
+  end
+
+  def take_photo
```

Comment:
your method clicks on element, it doesn't `take the photo` actually , name your methods according to what method does